test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -614,17 +618,13 @@ def get_vntype(self, mom):
         """
         Get the vntype if it exists for example on cray
         """
+        vntype = "no_cgroups"
         pbs_home = self.server.pbs_conf['PBS_HOME']
         vntype_f = os.path.join(pbs_home, 'mom_priv', 'vntype')
         self.logger.info("Reading the vntype value for mom %s" % mom)
-        if not self.du.isfile(hostname=mom, path=vntype_f):
-            return
-        with PBSLogUtils().open_log(
-                hostname=mom,
-                log=vntype_f,
-                sudo=True) as fd:
-            output = fd.read().splitlines()
-        return output
+        if self.du.isfile(hostname=mom, path=vntype_f):
+            output = self.du.cat(hostname=mom, filename=vntype_f, sudo=True)
+        return vntype
```

Comment:
You cat the file into output and then return vntype.  You used to return output.  Now output is not used.  Did you mean to change the return value?